htrc/torchlite/app.py

```diff
@@ -11,19 +14,29 @@
 from htrc.torchlite.filters import torchlite_stopword_filter, torchlite_stemmer, torchlite_lemmatizer
 from htrc.torchlite.middleware import TorchliteVersionHeaderMiddleware
 
-with open("config.yaml", mode="r", encoding="utf-8") as f:
-    config = safe_load(f)
+config_file: Optional[str] = os.getenv("TORCHLITE_CONFIG")
 
+if config_file is None:
+    raise OSError("Torchlite Configuration not found")
```

Comment:
I think [OSError](https://docs.python.org/3/library/exceptions.html#OSError) is used to indicate system-level error conditions, not application-level. Also, I would be more explicit about the problem... like "TORCHLITE_CONFIG not set -- cannot load Torchlite configuration"

Reply:
Probably better to implement a custom base exception like TorchliteError and have a derived class TorchliteConfigurationError or similar.

Reply:
2c9f0332cf6ab2eda6be1671e623ad57472446a1